client/src/components/main/auth_modal/AuthModal.js

```diff
@@ -42,7 +54,17 @@ export default ({ background })=> {
       </span>
     );
   } else {
-    headerContent = "Sign up";
+    headerContent = (
```

Comment:
Also why do we have the exact same message written in full in both the sign-up and sign-in conditionals? The whole point of having a conditional here was to give the user the context of the form, saying `Sign in or register` on both the sign in and sign up forms diminishes ease of use of the form.

Reply:
see above